tasks/st1105_primary_server_port_check.rb

```diff
@@ -33,6 +33,9 @@ def port_test(dest, port)
 end
 
 results_json = []
```

Comment:
Since this task uses an array of hashes for the result, we can add the deprecation message directly to `results_json` here.  Change lines 35-38 to be this:

```
results_json = [
  {
    'message' => 'This task is deprecated and will be removed in a future release. Please see this module’s README for more information.',
  },
]
```